.github/workflows/cypress-tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: Cypress tests
+on:
+  pull_request:
+    paths:
+      - 'packages/ui/**/*'
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
does it work with version 4 ? https://github.com/marketplace/actions/checkout

Reply:
it does!